docs/Release_Guide/release_steps/met/confirm_zenodo.rst

```diff
@@ -0,0 +1,15 @@
+Confirm Updated Zenodo Link
+---------------------------
+
+In the `MET GitHub repository <https://github.com/dtcenter/MET>`_
+scroll down to the README file and see:
+
+.. figure:: figure/met_zenodo.jpg
+
+Click on that to go to the MET Digital Object Identifier (DOI) on Zenodo.
```

Comment:
Julie, I think that the screenshot is probably overkill... especially since its not editable. If it ever changes we'd need to delete/replace that image. I think it's easier to just include a direct link in the release guide itself.
```suggestion
Check the [MET Digital Object Identifier (DOI)](https://doi.org/10.5281/zenodo.5565322) on Zenodo.
```

Reply:
Good point.  Sounds good.  Thank you.